### ISSUE TITLE
'Event' object has no attribute 'getMucroom' in will/plugins/chat_room/rooms.py

### ISSUE THREAD

**Issue by ptomkiel-oktawave** [skoczen/will#320]
It's fired by `room = self.get_room_from_message(message)`

full debug:
```
Traceback (most recent call last):
  File "/usr/lib64/python3.6/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib64/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File ".../lib/python3.6/site-packages/will/decorators.py", line 25, in wrapped_f
    f(*args, **kwargs)
  File ".../lib/python3.6/site-packages/will/plugins/chat_room/rooms.py", line 21, in participants_in_room
    room = self.get_room_from_message(message)
  File ".../lib/python3.6/site-packages/will/backends/io_adapters/hipchat.py", line 216, in get_room_from_message
    return self.get_room_by_jid(message.getMucroom())
AttributeError: 'Event' object has no attribute 'getMucroom'
```

You can try it out by enabling `will.plugins.chat_room` plugin and saying `@Will who is in this room?`

**Comment by skoczen** [comment]
@ptomkiel-oktawave  Nice catch, thanks!

Verified this issue on all platforms - this plugin was using an internal method, and I'll need to update it to the 2.x methods.   Is this blocking you?

**Comment by ptomkiel-oktawave** [comment]
Yes, I'm using same method in one of my plugins. If you can direct me to proper solution, I can wait for that :)

**Comment by TaunoTinits** [comment]
This worked for me: 
I replaced in file 
".../lib/python3.6/site-packages/will/backends/io_adapters/hipchat.py", line 216
    return self.get_room_by_jid(message.getMucroom())
with: 
    return self.get_room_from_name_or_id(message.data.channel.name)

**Comment by skoczen** [comment]
(Quick paste-on-all-open-issues update:)

**TL;DR**: New Will release EOM. Also, you're awesome.

**Full Update:** This may be TMI for you, and if so, cool, the thing you probably care about is above.  But given the total radio silence on my part - and the fact that I've always tried to create a community here where we got to be actual people instead of avatars and PRs, I wanted to give a bit more context.

As you may be aware from an earlier AFK, I got married in December, which was great, but burned the month from a Will perspective.  I planned to get back on it at the start of January, but instead found myself selling our house, and simultaneously hit with a surprisingly severe bout of depression.  The time since Jan 1 has been sort of a blur.  A dark blur, but a blur more than anything.   I saw the PRs and comments coming in, but didn't have the energy or time to do anything about them.

Things seem to be clearing up on both counts now, and I'm _really_ looking forward to getting these fixes in, catching up on bug reports, and being active in improving Will again. :)

With that, I'm aiming to get all outstanding PRs bundled into a release, and all issues replied to by the end of February.

Most of all, thanks for your contribution to Will, your patience with an imperfect, quite human project maintainer, and for continuing to come back, engage, and spend some of the precious seconds you get on earth making this project better.

I appreciate you.  Thank you.

**Comment by skoczen** [comment]
Hey @ptomkiel-oktawave and @TaunoTinits - thanks for your patience on this.  I'll be putting in @TaunoTinits 's fix in 2.1.1, which will be out shortly!

**Comment by skoczen** [comment]
This has been fixed, and released in 2.1.1!
